src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -57,6 +57,18 @@
       + "the connection string as hostname1:port1,hostname2:port2,hostname3:port3/chroot/path.";
   public static final String ZOOKEEPER_CONNECT_DEFAULT = "localhost:2181";
 
+  public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
+  private static final String
+      BOOTSTRAP_SERVERS_DOC =
+      "A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. "
+      + "The client will make use of all servers irrespective of which servers are specified here for bootstrapping—this "
+      + "list only impacts the initial hosts used to discover the full set of servers. This list "
+      + "should be in the form host1:port1,host2:port2,.... Since these servers are just used for "
```

Comment:
Was going to say we should clean this up since the `host1:port1` form doesn't cover the full format, though now I realize that this looks verbatim from the same Kafka config. We should probably update both since one of the major benefits of switching to the new consumer will be support for secure transports.